File: ProgrammersLifeiOS/ProgrammersLifeiOS/RepositoryContainer.m
//
//  RepositoryContainer.m
//  ProgrammersLifeiOS
//
//  Created by Zackary T Misso on 7/6/15.
//  Copyright © 2015 Zackary Misso. All rights reserved.
//

#import "RepositoryContainer.h"

@implementation RepositoryContainer

-(id)init{
    if(self = [super init]){
        // to be implemented
    }
    return self;
}

// getter methods
-(NSMutableArray*)getRepositories { return repositories; }

// setter methods
-(void)setRepositories:(NSMutableArray*)value { repositories = value; }

@end

File: ProgrammersLifeDemo/src/player/RepositoryContainer.java
// This is the Game Tester's version of ProgrammersLife
// Author :: Zackary Misso / FiralStudios
// Version :: 0.0.1

package player;
import java.util.ArrayList;
import dataStructures.Repository;

public class RepositoryContainer {
	private ArrayList<Repository> repositories;
	
	public RepositoryContainer(){
		repositories = new ArrayList<>();
	}
	
	// getter methods
	public ArrayList<Repository> getRepositories(){return repositories;}
	
	// setter methods
	public void setRepositories(ArrayList<Repository> param){repositories = param;}
}
